src/markdown-pages/custom-viz/configuration-options.mdx

```diff
@@ -29,7 +29,7 @@ _nr1.json_ is a metadata file that lives in your visualization directory and loo
 
 It contains the following top-level keys:
 
-- `schemaType`: Nerdpack artifacts all have `nr1.json` metadata files. The `schemaType` describes the artifact's schema. For all visualizations, `schemaType` is `VISUALIZATION`.
+- `schemaType`: Nerdpack nerdpack items all have `nr1.json` metadata files. The `schemaType` describes the nerdpack item's schema. For all visualizations, `schemaType` is `VISUALIZATION`.
```

Comment:
There are a few things I'd like to point out here:

- "Nerdpack" should always be capitalized in public documentation. 
- In the first sentence, you now have "Nerdpack nerdpack items". Let's change that to "Nerdpack items"
- In the second sentence, you can be less verbose. I recommend:

> - `schemaType`: Nerdpack items all have `nr1.json` metadata files. The `schemaType` describes the item's schema. For all visualizations, `schemaType` is `VISUALIZATION`.